bench/main.ts

```diff
@@ -31,24 +31,28 @@ for (const backend of backends) {
 
   let total = 0;
   let min!: number, max!: number;
-  for (let iter = 0; iter < ITERS; iter++) {
-    const now = performance.now();
-    const prep = backend.prepare("insert into test (value) values (?)");
-    for (let i = 0; i < ROWS; i++) {
-      prep.execute([`iter ${iter} loop ${i}`]);
-    }
-    prep.finalize();
-    const took = performance.now() - now;
-
-    if (min === undefined || max === undefined) {
-      min = max = took;
-    } else {
-      min = Math.min(min, took);
-      max = Math.max(max, took);
-    }
-
-    total += took;
-  }
+  Array.from(
```

Comment:
Why this over a simple `for`? Doesn't it also add a unnecessary array allocation? I think `for` loops are better in this case, maybe it might be even easier for V8 to JIT them.

Reply:
Simplicity can be subjective. I find it simpler because there's no setting up of what's being iterated over, just the array itself. However, performance reasons are well-justified and understable. `for` loops have slight performance advantage and would matter with larger amounts of data. Commit reverted.